Assets/Scripts/PlayerBase.cs

```diff
@@ -14,6 +14,13 @@ public class PlayerBase : MonoBehaviour
     private float m_xChange;
     private bool m_isJumping;
 
+    // [PN] TODO: make these constraints flexible depending on stage
+    private const float leftConstraint = -9.4f;
```

Comment:
maybe something like `Screen.width/height` if that's the full scene?

I'd like to have a good way to reason about tile units, screen resolution, etc, but I haven't yet written code like that.